Fix version and switch setup.py from distutils to setuptools to fix find packages
Author: Cintia Willemyns (Vrije Universiteit Brussel)
diff --git a/setup.py b/setup.py
index 2761c12..6eb62b6 100644
--- a/setup.py
+++ b/setup.py
@@ -1,10 +1,10 @@
-from distutils.core import setup
+from setuptools import setup, find_packages
 
 setup(name='rfdiffusion',
-      version='1.0.0',
+      version='1.1.0',
       description='RFdiffusion is an open source method for protein structure generation.',
       author='Rosetta Commons',
       url='https://github.com/RosettaCommons/RFdiffusion',
       scripts=["scripts/run_inference.py"],
-      packages=["rfdiffusion"],
-      install_requires=['torch', 'se3-transformer'])
\ No newline at end of file
+      packages=find_packages(),
+      install_requires=['torch', 'se3-transformer'])
